### PR TITLE
Added links for installing the tools

### PR DESCRIPTION
**Pull request by srclosson**
Just wanted to make it easier for readers/consumers to get to the installs if required.

### CODE REVIEW

File: docs/sources/operator/helm-getting-started.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ In this guide, you'll learn how to deploy [Grafana Agent Operator]({{< relref ".
 To deploy Agent Operator with Helm, make sure that you have the following:
 
 - A Kubernetes cluster
-- The `kubectl` command-line client installed and configured on your machine
-- The `helm` command-line client installed and configured on your machine
+- The `kubectl` command-line client [installed](https://kubernetes.io/docs/tasks/tools/#kubectl) and configured on your machine
```
Comment (osg-grafana) [inline docs/sources/operator/helm-getting-started.md:16]:
```suggestion
- The `[kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)` command-line client and configured on your machine
```

File: docs/sources/operator/helm-getting-started.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ In this guide, you'll learn how to deploy [Grafana Agent Operator]({{< relref ".
...
+- The `helm` command-line client [installed](https://helm.sh/docs/intro/install/) and configured on your machine
```
Comment (osg-grafana) [inline docs/sources/operator/helm-getting-started.md:17]:
```suggestion
- The `[helm](https://helm.sh/docs/intro/install/)` command-line client installed and configured on your machine
```

**Review by osg-grafana — APPROVED** [review]
Unblocking with nits

**Comment by osg-grafana** [comment]
Eek! Saw Helm and assumed it was my remit. Sorry about that. 🤦🏻

**Comment by tpaschalis** [comment]
@srclosson could you please sign the CLA so we can get this merged? Thanks!

**Review by clayton-cornell — APPROVED** [review]
Good idea to link

**Pull request merged**